CMake/config.cmake

```diff
@@ -134,10 +162,6 @@ write_basic_package_version_file(
 
 if (BUILD_INSTALLER)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/CMake/rttr-config-version.cmake"
-             DESTINATION cmake
+             DESTINATION ${CMAKE_INSTALL_DATADIR}/rttr/cmake
```

Comment:
Its the GNU Definition : https://cmake.org/cmake/help/v3.10/module/GNUInstallDirs.html
On windows it is set to `${CMAKE_INSTALL_PREFIX}`, since it does not really exist.

Reply:
so the cmake configuration mostly goes into `</usr/local>/share/rttr/cmake`. Most library do this...